apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -148,6 +127,54 @@ function LandingPage({
     />
   );
 
+  // Renders at most one banner for a user:
+  // - if the user hasn't used any PL resources yet, show the Getting Started banner
+  // - if the user has a pending workshop survery (mutually exclusive from the above), show a banner to fill out the survey
+  // - else, render either nothing or an announcement banner
+  const RenderBanner = () => {
+    const showGettingStartedBanner =
+      !currentYearApplicationId &&
+      !hasEnrolledInWorkshop &&
+      plCoursesStarted?.length === 0;
+
+    if (showGettingStartedBanner) {
+      return (
+        <TwoColumnActionBlock
+          imageUrl={pegasus(
+            '/images/fill-540x300/professional-learning/pl-superhero-girl-crop.png'
+          )}
+          heading={i18n.plLandingGettingStartedHeading()}
+          subHeading={i18n.plLandingGettingStartedSubHeading()}
+          description={i18n.plLandingGettingStartedDescription()}
+          buttons={[
+            {
+              url: pegasus('/educate/professional-learning'),
+              text: i18n.plLandingGettingStartedButton(),
+            },
+          ]}
+        />
+      );
+    } else if (lastWorkshopSurveyUrl) {
+      return RenderLastWorkshopSurveyBanner();
+    } else if (!!DCDO.get('curriculum-launch-2024', false)) {
+      return (
+        <TwoColumnActionBlock
+          imageUrl={pegasus(
+            '/images/fill-540x300/professional-learning/banner-books-with-background.png'
+          )}
+          subHeading={i18n.plLandingCurriculumLaunchBannerSubHeading()}
+          description={i18n.plLandingCurriculumLaunchBannerDescription()}
+          buttons={[
+            {
+              url: pegasus('/educate/professional-learning'),
+              text: i18n.plLandingCurriculumLaunchBannerButtonText(),
+            },
```

Comment:
question for reviewers: should this button have an aria label. It does only say "Start here" so not sure if we need to add more context

Reply:
I think technically it should, but since it's the only call to action in this section it's probs ok to not add one since the heading and description text clarifies where it goes.

But this button component should have an aria label prop, so it would be easy to add if you decide to!

Reply:
Added!